frontend/desktop/desktop.index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en", dir ="ltr">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simple Website <!-- will later be replaced--></title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body class="desktop">
+    <style>
+        @import url('https://fonts.cdnfonts.com/css/lucida-console');
+    </style>
```

Comment:
let's remove this `<style>` element && instead just add this line to the top of ur `frontend/desktop/styles.css`:
```css
@import url('https://fonts.cdnfonts.com/css/lucida-console');
```